api/classes/publisher.js

```diff
@@ -78,7 +78,7 @@ function upload(path, buffer, remixMetadata) {
   }
   var headers = {
     'Cache-Control': 'max-age=0',
-    'Content-Type': mimeType,
+    'Content-Type': mimeType + '; charset=utf-8',
```

Comment:
Small nit: officially it's `UTF-8`, rather than `utf-8`. Case doesn't always matter, so better to be safe.